docs/src/adr/009adr-outputs.md

```diff
@@ -67,9 +68,18 @@ following:
 
 ## 4. Global Configuration File
 
-Apalache should define a global configuration file `apalache-global-config`, e.g. in the `<HOME>/.tlaplus` directory, in which users can define the default values of all parameters, including all flags listed in section 1, as well as `out-dir`. The format of the configuration file is an implementation detail and will not be specified here.
+Apalache should define a global configuration file `apalache.cfg`, e.g. in the `<HOME>/.tlaplus` directory, in which users can define the default values of all parameters, including all flags listed in section 1, as well as `out-dir`. The format of the configuration file is an implementation detail and will not be specified here.
```

Comment:
I recall Igor complaining about the name `apalache.cfg` last time I suggested it
https://github.com/informalsystems/apalache/pull/1025#discussion_r728696302